doc/devel/saved_replies.rst

```diff
@@ -0,0 +1,95 @@
+to support our community with.
+
+As a maintainer, triage team member, or any other active community member, it may be helpful to store some of these in
+your GitHub account’s `saved replies`_ for reviewing issues and PRs, and supporting other community members, as they participate
+and contribute to Matplotlib.
+
+Our saved replies can be long and detailed. This is because we aim to offer support without assuming the level of
+experience, or the intent the community member we are addressing might have. There is still room for one to tweak and
+personalize the message, if based on the ongoing conversation some parts are redundant, require further information, or
+if they want to add a personal note to suggest that they will be available to offer support, for example.
+
+Our intent with providing a baseline is to make it easier to not miss the minimum amount of information, and
+consistently facilitate the respectful and welcoming interactions we aspire to maintain, not to set rigid communication
+constraints. We trust our maintainers, triage team members, and all community members who continue to abide by our
+`Code of Conduct`_, to communicate freely. It is based on their ongoing experience that we are able to provide these saved
+replies to begin with.
+
+.. _Code of Conduct: https://github.com/matplotlib/matplotlib/blob/master/CODE_OF_CONDUCT.md
+.. _saved replies: https://github.com/settings/replies/
+
+Issues
+------
+
+Coming soon 😉
+
+Pull Requests
+-------------
+
+First Pull Request Merged
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Hi-five ✋ on merging your first pull request to Matplotlib, @username! We hope you stick around and invite you to continue to take an active part in Matplotlib!
+
+Your choices aren’t limited to programming 😉 – you can review pull requests, help us stay on top of new and old issues, develop educational material, refresh our documentation, work on our website, create marketing materials, translate website content, write grant proposals, and help with other fundraising initiatives. For more info, check out: https://matplotlib.org/stable/devel/index
+
+Also, consider joining our `mailing list`_ ✉️. This is a great way to connect with other people in our community and be part of important conversations that affect the development of Matplotlib.
+
+If you haven’t yet, do join us on `gitter`_ and `discourse`_ 🗣. The former is a chat platform, which is great when you have any questions on process or how to fix something, the latter is a forum which is useful for longer questions and discussions.
+
+Last but not least, we have a monthly meeting 👥 for new contributors and a weekly meeting for the maintainers, everyone is welcome to join both! You can find out more about our regular project meetings in this `calendar page`_.
+
+.. _mailing list: https://mail.python.org/mailman/listinfo/matplotlib-devel
+.. _discourse: https://discourse.matplotlib.org/
+.. _calendar page: https://scientific-python.org/calendars/
```

Comment:
In my opinion, taking the time to say more on the PR creates a different experience and we should offer that (I would like to do that personally). It's a moment where one has already done more than just say they want to be a part. They took part. And signaling that we have a few more things to offer, in an interaction that isn't available to those who haven't reached this step, is respectful of their contribution. Even if all of this information is on the contributor guide.

Reply:
I think that there is a risk when leaving a "personal" message that is quite long. At some stage it will be obvious that it is not "personal" anymore and may work oppositely to the purpose. So I'd also be slightly in favor of pointing elsewhere.

Reply:
I beg to differ. I think taking the time to notice people's contributions is attentive. If it was an automatic bot, that's impersonal. 

If you think the length is of particular concern, then I would appreciate to understand why that would be off-putting to new contributors, so that I can edit it down appropriately. I can maybe offer only one of the communication channel, and then link for more options for example.
But I feel like I am lacking context on the length of a message being a problem to begin with and would appreciate insight.